src/main/java/com/telefonica/iot/cygnus/sinks/OrionMongoSink.java

```diff
@@ -51,7 +51,12 @@ void persist(Map<String, String> eventHeaders, NotifyContextRequest notification
         // get some header values
         Long recvTimeTs = new Long(eventHeaders.get("timestamp"));
         String fiwareService = eventHeaders.get(Constants.HEADER_SERVICE);
-        String fiwareServicePath = eventHeaders.get(Constants.HEADER_SERVICE_PATH);
+        String[] fiwareServicePaths = eventHeaders.get(Constants.HEADER_SERVICE_PATH).split(",");
```

Comment:
This is hard to explain, but I'll try :) Originally, an Orion notification, which may contain serveral context elements, has a unique fiware-servicePath for all of them. Nevertheless, internally to Cygnus this common fiware-servicePath may be discarded in favor of an alternative fiware-servicePath for each context element if the pattern-based grouping mechanisms is configured to do that. For instance, a notification may contain the "speed" value for a "car.1" of type "car" and the "speed" value for a "car.mycar" of type "car"; the common fiware-servicePath is "/4wheels". If the following rules are configured:

* "all cars called as *.d will go to a destination called numeric_cars in a fiware-servicePath called /4wheels_numeric_cars"
* "all cars called as *.D will got to a destination called other_cars in a fiware-servicePath called /4wheels_other_cars"

Then we have the origianl fiware-servicePath transformed in a list of two elements, containing ["/4wheels_numeric_cars","/4wheels_other_cars"].